CGBEmu/Player.h

```diff
@@ -1,24 +0,0 @@
-#include <iostream>
```

Comment:
There was already a Player file in the source directory. this file was a copy that was created in one folder higher. As this file lived in the wrong location and had the wrong format, I removed it.